UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -633,13 +629,13 @@ private void ValidateHideFlags()
 
 		// Need to check all flags with their gameobject names...
 		// TODO: it should be done much easier
-		ValidateHideFlag(ClothingHideFlags.HIDE_GLOVES, "hands");
-		ValidateHideFlag(ClothingHideFlags.HIDE_JUMPSUIT, "uniform");
-		ValidateHideFlag(ClothingHideFlags.HIDE_SHOES, "feet");
-		ValidateHideFlag(ClothingHideFlags.HIDE_MASK, "mask");
-		ValidateHideFlag(ClothingHideFlags.HIDE_EARS, "ear");
-		ValidateHideFlag(ClothingHideFlags.HIDE_EYES, "eyes");
-		ValidateHideFlag(ClothingHideFlags.HIDE_NECK, "neck");
+		// ValidateHideFlag(ClothingHideFlags.HIDE_GLOVES, "hands");
```

Comment:
nani?

Reply:
I can't see anything that uses them, from a quick look through it looks like it just goes through all the clothing and then if any have the flag then it hides it, so the names are used nowhere?